src/Razor/test/testapps/AppWithP2PReference/AppWithP2PReference.csproj

```diff
@@ -6,6 +6,7 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.0</TargetFramework>
+    <_ReferencesMicrosoftAspNetCoreApp>true</_ReferencesMicrosoftAspNetCoreApp>
```

Comment:
Suggest putting this in a Directory.Build.props file or files if possible.

Reply:
Unfortunately some of these projects don't reference the shared fx (like the ones that specifically reference Components or reference 2.1 MVC). Moving it to the root is a bit tricky